src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp

```diff
@@ -225,6 +225,11 @@ class Minmod<VolumeDim, tmpl::list<Tags...>> {
     tuples::TaggedTuple<Minmod_detail::to_tensor_double<Tags>...> means_;
     std::array<double, VolumeDim> element_size_ =
         make_array<VolumeDim>(std::numeric_limits<double>::signaling_NaN());
+
+    void pup(PUP::er& p) noexcept {
```

Comment:
You'll need a `NOLINT` comment to silence clang-tidy

Reply:
Looks like clang-tidy passes on Travis... should I add the comment anyway?

Reply:
Yes, that just means you're not using this code in a cpp file